docs/Install-Guide/Install.md

```diff
@@ -11,16 +11,16 @@ Packages are provided according to this [table](./Community-Packages.md).
 
 ###### For Debian
 
-Add the GPG key to apt:
+Download the GPG key to apt config directory:
 
 ```console
-wget -O - https://download.gluster.org/pub/gluster/glusterfs/9/rsa.pub | apt-key add -
+wget -O - https://download.gluster.org/pub/gluster/glusterfs/9/rsa.pub | gpg --dearmor > /etc/apt/keyrings/gluster.gpg
```

Comment:
Store the keyfile here: /etc/apt/trusted.gpg.d

This should be default to debian.

Reply:
Done :heavy_check_mark: